fixieai/agents/code_shot.py

```diff
@@ -169,7 +169,7 @@ def func(query):
 
     def _handshake(self) -> fastapi.Response:
         """Returns the agent's metadata in YAML format."""
-        metadata = AgentMetadata(self.base_prompt, self.few_shots)
+        metadata = AgentMetadata(self.base_prompt, self.few_shots)  # type: ignore[call-arg]
```

Comment:
OOC, what are all these ignores for?

Reply:
For whatever reason mypy does not seem to understand that the dataclasses library creates a constructor for the class with these parameters, and complains about this call. I did some Googling and wasn't able to find a solution other than to suppress the error. If you happen to know I'm happy to do something different and get rid of these!